link-dependencies/README.md

```diff
@@ -0,0 +1,27 @@
+# 🛠️ `link-dependencies`
+
+Attempts to add links to the original changelogs for dependency bumps in changelog.yaml.
+
+The link is computed automatically when the dependency name is a full route, like `github.com/org/package`, or it's got from a [dictionary file](../README.md#dictionary-file) when present.
+
+## Example Usage
+
+Example with a dictionary:
+```yaml
+- name: Link dependencies
+  uses: newrelic/release-toolkit/link-dependencies@v1
+  with:
+    dictionary: .github/dictionary.yaml
+```
+
+Dictionary file:
+
+```yaml
+dictionary:
+  common-library: "https://github.com/newrelic/helm-charts/releases/tag/common-library-{{.To}}"
```

Comment:
is it To or To.Original?

Reply:
It depends on each case, because some github repos have v1.1.1 and others have 1.1.1 the original will respect the v in case they have it.